.gitignore

```diff
@@ -17,7 +17,8 @@ vendor
 
 # Test binary, build with `go test -c`
 *.test
-test/fixtures/keys
+# Generate test PKIs with `go generate`
+pkg/didcomm/transport/http/testdata
```

Comment:
I would put the generated certs in a subdirectory of testdata and only ignore that directory.

Reply:
this means testdata will be committed to git...

Reply:
done, new subdirectory called crypto/ is added